src/main/java/com/kodilla/ecommercee/domain/OrderEntity.java

```diff
@@ -22,5 +23,24 @@
     @GeneratedValue
     private long id;
 
+    @GeneratedValue
+    @NotNull
+    private int orderNumber;
+
     private String productGroup;
+    private int price;
```

Comment:
Jak wyżej co do ceny :)

Reply:
jeśli byłby to bank to rekomendowałbym BigDecimal. tutaj może jednak precyzja double wystarczy

Reply:
Zmieniłem na BigDecimal